.github/renovate.json5

```
@@ -29,6 +29,10 @@
       "matchPackagePrefixes": ["com.linecorp.armeria:"],
       "groupName": "armeria packages"
     },
+    {
+      "matchPackagePrefixes": ["com.diffplug.spotless:"],
```

I'm not sure if this will pick up #9576 since package name is exactly `com.diffplug.spotless`

```suggestion
      "matchPackagePrefixes": ["com.diffplug.spotless"],
```